aiven/templates/service_user_config_schema.json

```diff
@@ -1278,24 +1598,34 @@
         "default": null,
         "title": "This setting is deprecated. Use read-replica service integration instead.",
         "example": true,
-        "type": ["boolean", "null"]
+        "type": [
+          "boolean",
+          "null"
+        ]
       },
       "pg_service_to_fork_from": {
         "createOnly": true,
         "default": null,
         "example": "anotherservicename",
         "title": "Name of the PG Service from which to fork (deprecated, use service_to_fork_from). This has effect only when a new service is being created.",
-        "type": ["string", "null"]
+        "type": [
+          "string",
+          "null"
+        ]
       },
       "pg_version": {
-        "enum": ["9.5", "9.6", "10", "11"],
+        "enum": [
+          "9.5",
+          "9.6",
+          "10",
+          "11",
+          "12"
+        ],
         "title": "PostgreSQL major version",
-        "type": ["string", "null"]
-      },
-      "variant": {
```

Comment:
This shouldn't be dropped. See internal pull request related to this.

Reply:
It is not dropped, the automatic schema generation script just moved it to an other location (line 1723)